### PR TITLE
Improve the docs

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1307, #1297

* improve config
* improve local config
* improve themes
* add SVG section

### CODE REVIEW

File: configs/local.sample.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
-  "firefox": {
-    "geckoDir": "/Users/jlaster/src/mozilla/gecko"
```
Comment (clarkbw) [inline configs/local.sample.json]: Much better!  This sample never helped me.

File: docs/local-development.md [hunk 2]
```diff
@@ -94,6 +87,116 @@ html[dir="rtl"] .source-footer .command-bar {
...
+We use SVGs in DevTools because they look good at any resolution.
+
+
+
+**Adding a new SVG**
+
+* add the SVG in [assets/images](../assets/images)
+* add it to [Svg.js](../assets/images/Svg.js)
+
+```diff
```
Comment (clarkbw) [inline docs/local-development.md]: I like the diffs as helpers, this is a great system.

File: packages/devtools-local-toolbox/src/components/Root.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 const classnames = require("classnames");
 const { getValue, isDevelopment } = require("devtools-config");
 
+require("./Root.css");
```
Comment (clarkbw) [inline packages/devtools-local-toolbox/src/components/Root.js]: This looks like a separate thing from the docs changes

Comment (jasonLaster) [inline packages/devtools-local-toolbox/src/components/Root.js]: yeah, turns out that it wasn't being pulled in so the dark theme scrollbar looked pretty bad in windows.

**Pull request merged**